packages/block-library/src/cover/edit/inspector-controls.js

```diff
@@ -73,15 +73,16 @@ function CoverHeightInput( {
 
 	return (
 		<UnitControl
-			label={ __( 'Minimum height of cover' ) }
+			label={ __( 'Minimum height' ) }
 			id={ inputId }
 			isResetValueOnUnitChange
 			min={ min }
 			onChange={ handleOnChange }
 			onUnitChange={ onUnitChange }
-			__unstableInputWidth="80px"
+			__unstableInputWidth="calc(50% - 4px)"
```

Comment:
Those 4px make up half the 8px gap that sits between width/height in the image block. There's a better way to do this, but is it this PR? I can also just type in 120px.

Reply:
I think that 8px may be incorrect 🙈 

According to https://github.com/WordPress/gutenberg/issues/46734 the gap between columns should be 16px. This is the case in the background size inputs:

<img width="269" alt="Screenshot 2024-06-21 at 16 15 38" src="https://github.com/WordPress/gutenberg/assets/846565/d18ef322-6d21-425f-8c86-2e5c0d32495a">

Ideally we avoid a hard-coded value, just in case the inspector width changes in the future.

Reply:
Probably, there is a difference in the gap values ​​between the two, so in the future it will be necessary to unify the gap rules.

`ToolsPanel` has a gap value of `8px` ([Source](https://github.com/WordPress/gutenberg/blob/0384b0ee3f57fbf77038e786f08c254a0e1b27ea/packages/components/src/tools-panel/tools-panel/component.tsx#L33)):

![tools-panel](https://github.com/WordPress/gutenberg/assets/54422211/24541102-dc42-4a68-9690-c9b0987e4260)

On the other hand, the `FocalPointPicker` component has a gap value of `16px` ([Source](https://github.com/WordPress/gutenberg/blob/0384b0ee3f57fbf77038e786f08c254a0e1b27ea/packages/components/src/focal-point-picker/controls.tsx#L57)):

![focal-point-picker](https://github.com/WordPress/gutenberg/assets/54422211/66f2e2f7-2f09-4b3c-a428-5e49ce0ade37)

Which one should we unify it to? I would lean towards standardizing on `8px`.

By the way, width/height in the image block, `grid-column: span 1` is applied:

https://github.com/WordPress/gutenberg/blob/0384b0ee3f57fbf77038e786f08c254a0e1b27ea/packages/block-editor/src/components/dimensions-tool/width-height-tool.js#L15-L17

Therefore, I think the following approach is also possible:

<details><summary>Details</summary>

```diff
diff --git a/packages/block-library/src/cover/edit/inspector-controls.js b/packages/block-library/src/cover/edit/inspector-controls.js
index 39dd425c62..65c32fd526 100644
--- a/packages/block-library/src/cover/edit/inspector-controls.js
+++ b/packages/block-library/src/cover/edit/inspector-controls.js
@@ -79,7 +79,6 @@ function CoverHeightInput( {
                        min={ min }
                        onChange={ handleOnChange }
                        onUnitChange={ onUnitChange }
-                       __unstableInputWidth="calc(50% - 4px)"
                        units={ units }
                        value={ computedValue }
                        size="__unstable-large"
@@ -314,6 +313,7 @@ export default function CoverInspectorControls( {
                                        } ) }
                                        isShownByDefault
                                        panelId={ clientId }
+                                       style={ { gridColumn: 'span 1' } }
                                >
                                        <CoverHeightInput
                                                value={
```

</details>

Reply:
I think the intended value is 16px, and there's a discrepancy here. I'm happy to take a stab at trying to unify it here. 

![Screenshot 2024-06-24 at 09 47 08](https://github.com/WordPress/gutenberg/assets/1204802/19f5f918-0289-41ef-ab80-3fde5b706197)

I have a feeling this might be best done separately, so what do you think I should set as a value for this one in the mean time?

Reply:
I noticed that there is a style for a single column of the dimension panel:

https://github.com/WordPress/gutenberg/blob/688ff13706638ce0174fff396375468cadc63920/packages/block-editor/src/hooks/dimensions.scss

Since the min-height control is also part of the dimension panel, we should be able to make the following changes to this PR:

```diff
diff --git a/packages/block-library/src/cover/edit/inspector-controls.js b/packages/block-library/src/cover/edit/inspector-controls.js
index 39dd425c62..80110d5158 100644
--- a/packages/block-library/src/cover/edit/inspector-controls.js
+++ b/packages/block-library/src/cover/edit/inspector-controls.js
@@ -79,7 +79,6 @@ function CoverHeightInput( {
                        min={ min }
                        onChange={ handleOnChange }
                        onUnitChange={ onUnitChange }
-                       __unstableInputWidth="calc(50% - 4px)"
                        units={ units }
                        value={ computedValue }
                        size="__unstable-large"
@@ -300,6 +299,7 @@ export default function CoverInspectorControls( {
                        ) }
                        <InspectorControls group="dimensions">
                                <ToolsPanelItem
+                                       className="single-column"
                                        hasValue={ () => !! minHeight }
                                        label={ __( 'Minimum height' ) }
                                        onDeselect={ () =>
```

The width value ​​is not hard-coded, so when the `ToolsPanel` gap changes from `8px` to `16px`, we don't need to update the min-height control's style.

Reply:
Nice, that works!
![Screenshot 2024-06-24 at 10 51 51](https://github.com/WordPress/gutenberg/assets/1204802/7f4c1d7a-5cdc-4435-a419-9396883588b6)

You'll see that it doesn't line up with the border controls below, but as you surfaced with width/height and left/top, that's a separate thing to fix!